stream-chat-android-ui-components/src/main/res/layout/stream_ui_item_message_giphy.xml

```diff
@@ -95,7 +95,7 @@
                 app:layout_constraintLeft_toRightOf="@id/giphyTextLabel"
                 app:layout_constraintRight_toRightOf="parent"
                 app:layout_constraintTop_toTopOf="@id/previousButton"
-                app:tint="@color/stream_ui_icon_strong_tint"
+                app:tint="#000000"
```

Comment:
These button colors here are hardcoded and non-named in the design as well, and they don't change between light and dark modes